modal/cli/_traceback.py

```diff
@@ -180,10 +181,16 @@ def showwarning(warning, category, filename, lineno, file=None, line=None):
             except OSError:
                 # e.g., when filename is "<unknown>"; raises FileNotFoundError on posix but OSError on windows
                 pass
+            if issubclass(category, ServerWarning):
+                title = "Modal Warning"
```

Comment:
could make this a property of the class (or an abstract method!) so you don't have to check the type

Reply:
Yeah not a bad idea. We've talked a little bit about other useful metadata that we could add to our exceptions (e.g. surfacing the ID of the modal object that an exception is from / about).